src/main/kotlin/org/rucca/cheese/model/KnowledgeDelete200ResponseDTO.kt

```
@@ -0,0 +1,17 @@
+/**
+ * @param code
+ * @param message
+ */
+data class KnowledgeDelete200ResponseDTO(
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("code", required = true)
+    val code: kotlin.Int = 0,
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("message", required = true)
+    val message: kotlin.String,
+) {}
```

detekt (1.23.7)

[warning] The class or object KnowledgeDelete200ResponseDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)